template.go

```diff
@@ -240,3 +297,14 @@ func (t *Templater) getTemplateByPath(path string) *File {
 	}
 	return nil
 }
+
+// pathIsIgnored provides a quick way to check whether a given path should be
+// ignored based on the 'IgnoreRegex' field.
+func (t *Templater) pathIsIgnored(filePath string) bool {
```

Comment:
Would rename this to `IsPathIgnored`, to keep the convention of having the verb first.

http://codelegance.com/semantic-method-naming/

Reply:
You're right, renamed 🚀